docker/nightly/setup.python.sh

```
@@ -0,0 +1,55 @@
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+#
+# setup.python.sh: Install a specific Python version and packages for it.
+# Usage: setup.python.sh <pyversion> <requirements.txt>
+set -xe
+
+source ~/.bashrc
```

Shellcheck

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)